sdk/helper/awsutil/region.go

```diff
@@ -37,22 +36,21 @@ Our chosen approach is:
 
 This approach should be used in future updates to this logic.
 */
-func GetOrDefaultRegion(logger hclog.Logger, configuredRegion string) string {
+func GetRegion(configuredRegion string) (string, error) {
 	if configuredRegion != "" {
-		return configuredRegion
+		return configuredRegion, nil
 	}
 
 	sess, err := session.NewSessionWithOptions(session.Options{
 		SharedConfigState: session.SharedConfigEnable,
 	})
 	if err != nil {
-		logger.Warn(fmt.Sprintf("unable to start session, defaulting region to %s", DefaultRegion))
-		return DefaultRegion
+		return "", fmt.Errorf("got error when starting session: %w", err)
```

Comment:
Replace with errwrap?

Reply:
This is breaking CI because we're still using go 1.12.

Reply:
We shouldn't be -- 1.4 will require 1.13, we should be on it already.

Reply:
Changed.